doc-content/enterprise-only/optimizer/business-optimizer-test-application-quarkus-proc.adoc

```diff
@@ -0,0 +1,101 @@
+
+[id='business-optimizer-test-application-proc_{CONTEXT}']
```

Comment:
Note: there a new subsection here in the OptaPlanner Quarkus guide for ConstraintVerifiers: https://github.com/quarkusio/quarkus/pull/15611 (PR still open; waiting for merge)

Reply:
Will this affect the 7.11 release, @chrisopher-Chianelli?

Reply:
What version of OptaPlanner is in the 7.11 release? It affects 8.5.0.Final and above

Reply:
I'm not sure, @christopher-Chianelli but if it hasn't been merged then I assume it won't be in 7.11 because we just got CR1.

Reply:
The feature been merged, the doc wasn't (up to Quarkus maintainers)